schedule/schedule-appengine/src/main/client/jasify/admin/activity/admin-activity.js

```diff
@@ -2,10 +2,21 @@
 
     angular.module('jasifyWeb').controller('AdminActivityController', AdminActivityController);
 
-    function AdminActivityController($location, ActivityType, Activity, activity, organizations) {
+    angular.module('jasifyWeb').directive('datepickerPopup', function () {
+        return {
+            restrict: 'EAC',
+            require: 'ngModel',
+            link: function (scope, element, attr, controller) {
+                //remove the default formatter from the input directive to prevent conflict
+                controller.$formatters.shift();
+            }
+        };
+    });
+
```

Comment:
* directive name should start with jas (**jas**DatepickerPopup)  so that we don't clash with others.
 * directive should be in a dedicated file `directives/datepicker-popup.directive.js` (copy the *boylerplate* from another directive)
```javascript
(function (angular) {
  //code here
})(angular);
```